### ISSUE TITLE
[HACKTOBERFEST 2024] - Audio and Video Quiz

### ISSUE THREAD

**Issue by naomi-lgbt** [freeCodeCamp/freeCodeCamp#56212]
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the Audio and Video quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-audio-and-video`.

**All quiz questions need to relate to one of these topics:**

- Audio constructor
- play(), pause(), stop()
- video and audio formats
- `source` element hand handling multiple formats
- codecs
- HTMLMediaElement API
- media streams, Media Stream Recording API, and getUserMedia()
- Web Audio API, Media Source Extensions API

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

**Comment by naomi-lgbt** [comment]
Thank you for your interest in contributing. This issue will be opened for contribution when Hacktoberfest starts on October 1st.

> [!CAUTION]
> Pull requests which are **created before** October 1st will not be eligible for Hacktoberfest. We appreciate enthusiasm, but encourage you to hold off on contributing to this issue until the event has begun.

If you wish to be notified when this issue is opened for contribution, subscribe to notifications:

![image](https://github.com/user-attachments/assets/a1f1f2f5-8c64-403a-a5be-5d5495d8f886)

**Comment by naomi-lgbt** [comment]
# This issue is now open
If you would like to work on this issue, please leave a comment so we can assign it to you.

[Be sure to read our contributing guidelines](https://contribute.freecodecamp.org).

**Comment by nayanmapara** [comment]
Hi @naomi-lgbt , I would love to contribute to this issue! Could you please assign it to me?

**Comment by prachidoddi** [comment]
I'd like to help with this issue. Could you please assign it to me?

**Comment by gikf** [comment]
Hey @nayanmapara,

you are already assigned to different issue - https://github.com/freeCodeCamp/freeCodeCamp/issues/56208.

**Comment by Code-Hacker26** [comment]
Hey @naomi-lgbt I want to work on this issue

**Comment by jdwilkin4** [comment]
@Code-Hacker26 

You have already been assigned to another issue 
https://github.com/freeCodeCamp/freeCodeCamp/issues/56185

**Comment by AnarchistHoneybun** [comment]
Hi, I'd like to claim this if possible?

**Comment by AnarchistHoneybun** [comment]
@naomi-lgbt I don't see this file `/curriculum/challenges/english/25-front-end-development/quiz-audio-and-video` in the repo. Other issues I checked out seemed to have their files already in place, so was just wondering if the expectation is to create the file in my own fork?
related: what goes in the `id` frontmatter field of the quiz sheet

**Comment by jdwilkin4** [comment]
Here is the file you need to use  @AnarchistHoneybun 

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/25-front-end-development/quiz-javascript-audio-and-video/66edd3b3096349f06cf688bb.md

**Comment by naomi-lgbt** [comment]
> Other issues I checked out seemed to have their files already in place, so was just wondering if the expectation is to create the file in my own fork?

Sorry about that - I automated the directory name generation, and a couple of them aren't 1:1 pairs with the actual file structure.

**Comment by naomi-lgbt** [comment]
# This issue has been assigned a contributor. To prevent extraneous notifications, the issue is now locked. If you have any questions, please reach out to me in our [Discord community](https://chat.freecodecamp.org).
